Basic-Car-Maintenance/Shared/BasicCarMaintenanceApp.swift

```diff
@@ -46,7 +46,7 @@ class AppDelegate: NSObject, UIApplicationDelegate {
         
         FirebaseApp.configure()
         
-        let useEmulator = UserDefaults.standard.bool(forKey: "useEmulator")
+        let useEmulator = UserDefaults.shared.bool(forKey: "useEmulator")
```

Comment:
what does this do? changing the `UserDefaults` to use the shared app group?

Reply:
I did this so the widget would respect the same "useEmulator" value as the main app.